.gitignore

```diff
@@ -0,0 +1,33 @@
+.Rhistory
+.Rapp.history
+.RData
+.Rproj.user/
+*.RData
+.Renviron
+.Rprofile
```

Comment:
**issue:** We actually want to include the default `.Rprofile` so that `renv` successfully activates.